packages/atomic/src/locales.json

```diff
@@ -226,10 +226,6 @@
     "zh-cn": "每一页的结果",
     "zh-tw": "每頁的結果數"
   },
-  "display-results-per-page": {
-    "en": "Display {{results}} results per page",
-    "fr": "Afficher {{results}} résultats par page"
-  },
```

Comment:
I think this is redundant, since the parent `radiogroup` already describes this as being "Results per page" and there's a label preceding the radiogroup.